packaging/homebrew/README.md

```
@@ -113,9 +113,94 @@ brew audit --strict --formula ./packaging/homebrew/apispec.rb
+| Open-source with an OSI licence | Apache-2.0 |
+| Stable, versioned releases | v0.5.6, tagged and signed |
+| Maintained, not a duplicate of an existing formula | yes |
+| **Builds from source** | this is the work — see below |
+
+### The one blocking difference
+
+core formulae **build from source**; the tap formula downloads a pre-built
+binary, which core does not accept. `apispec-homebrew-core.rb` in this directory
+is the source-building version, verified locally:
+
+```bash
```

markdownlint-cli2 (0.23.2)

[warning] Code block style
Expected: indented; Actual: fenced

(MD046, code-block-style)